scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/JobsView.java

```diff
@@ -131,7 +131,7 @@ protected Widget getBuiltFilterPane() {
 
     @Override
     protected Label getLabel() {
-        Label label = new Label("Use filters to restrict the number of jobs currently displayed.<br>");
+        Label label = new Label("Use filters to restrict the number of jobs currently displayed.<br>Be aware that <b> using multiple filters on the same criteria</b> (id, priority,..) lead in most cases to <b>only consider your last filter</b>.");
```

Comment:
Is that true? I thought you fixed the issue by adding your tricky RETURN_NOTHING_FILTER input filter

Reply:
RETURN_NOTHING_FILTER is only used to consider combination of filters returning empty result (id=3 && id=4, name="toto" && name="tata", ..). But for other combinations of filters on the same criteria (which should return non empty result), since the AND operator is not supported in our current graphql implementation, we decided to only consider the last filter (rather than return ing void). And this is why we updated the information message.

Reply:
I see I see. Thanks